Assignment09Starter/part01/process.php

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 1</title>
+</head>
+<body>
+
+<?php
+
+
+
+    $weight = $_POST['weight'];
+    $height = $_POST['height'];
+    
+       { $bmi = $weight / ($height * $height);
```

Comment:
here you should be defining a function - which is part of what we'll go over today in our zoom.